src/components/Registers.js

```diff
@@ -1,20 +1,33 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-// import Category from './Category';
+import React, { useState } from 'react';
+// import PropTypes from 'prop-types';
 import RegMenu from './RegMenu';
-// import {
-//   Pointers,
-//   Temporary,
-//   CalleeSaved,
-//   Arguments,
-// } from '../utils/constants';
+// import useCPU from '../hooks/use-cpu';
+import useInterval from '../hooks/use-interval';
+
 
 import getRegisters from '../utils/registers';
 
-const RegisterPanel = ({ registers }) => {
+const RegisterPanel = () => {
```

Comment:
could we rename the file to RegisterPanel.js, it's a convention to give the filename the same as the component name

Reply:
done